src/check_python_versions/sources/poetry_pyproject.py

```diff
@@ -38,11 +39,29 @@
 PYPROJECT_TOML = 'pyproject.toml'
 
 
+def _load_toml(filename: FileOrFilename):
```

Comment:
Would be nice if there was a type annotation for the return type.  I suppose it's hard to be more specific than a generic `Dict[str, Any]`.

Reply:
seems this row is outdated, also the file was renamed to "pyproject.py" because it handles all the tools that use pyproject.toml